USCoreAliases.fsh

```diff
@@ -12,6 +12,6 @@ Alias:   USCoreObservationLab = http://hl7.org/fhir/us/core/StructureDefinition/
 Alias:   USCoreOrganization = http://hl7.org/fhir/us/core/StructureDefinition/us-core-organization
 
 // Extensions
-Alias:   USCoreRace = http://hl7.org/fhir/us/core/StructureDefinition/us-core-race
+Alias:   USCoreRace = urn:oid:2.16.840.1.113883.6.238
```

Comment:
1) This file has aliases for US Core StructureDefinitions. For consistency, I would leave it as it was. It might not be necessary to define the alias, but it was consistent.
2) Add the code system alias to the file `CodeSystemAliases.fsh` as follows:
`* Alias:  OmbRaceCat = urn:oid:2.16.840.1.113883.6.238
3) Change the example to reflect the code system alias:
`* extension[race].extension[ombCategory].valueCoding = OmbRaceCat#1002-5 "American Indian or Alaska Native"`